config/processCcd.py

```diff
@@ -10,3 +11,13 @@
                                        'POSTSECA', 'POSTSECB']
 
 config.charImage.repair.cosmicray.nCrPixelMax = 100000
+
+for refObjLoader in (config.calibrate.astromRefObjLoader,
+                     config.calibrate.photoRefObjLoader,
+                     config.charImage.refObjLoader,
+                     ):
+    refObjLoader.retarget(LoadIndexedReferenceObjectsTask)
+    refObjLoader.ref_dataset_name = "ps1_pv3_3pi_20170110"
+    refObjLoader.filterMap['u'] = 'g'
```

Comment:
I do not have any color term data though. In this case is it better to leave out u-band?